src/core/Akka.Streams.Tests/Dsl/FlowFlattenMergeSpec.cs

```diff
@@ -41,14 +41,14 @@ public FlowFlattenMergeSpec(ITestOutputHelper helper) : base(helper)
         private Sink<int, Task<ImmutableHashSet<int>>> ToSet =>
                 Flow.Create<int>()
                     .Grouped(1000)
-                    .Map(x => x.ToImmutableHashSet())
+                    .Select(x => x.ToImmutableHashSet())
                     .ToMaterialized(Sink.First<ImmutableHashSet<int>>(), Keep.Right);
 
         [Fact]
         public void A_FlattenMerge_must_work_in_the_nominal_case()
         {
             var task = Source.From(new[] {Src10(0), Src10(10), Src10(20), Src10(30)})
-                .FlatMapMerge(4, s => s)
+                .MergeMany(4, s => s)
```

Comment:
Not quite since `ConcatMany` (known as old `FlatMapConcat`) also uses that class.